### PR TITLE
feat(cozy-doctypes): Add getDisplayName for contacts

### PR DESCRIPTION
**Pull request by cedricmessiant**
Use fullname if there is any, else use primary email

### CODE REVIEW

File: packages/cozy-doctypes/src/contacts/Contact.spec.js [hunk 1]
```diff
@@ -102,4 +102,42 @@ describe('Contact model', () => {
...
+            address: 'doran.martell@dorne.westeros',
+            primary: true
+          }
+        ]
+      }
+      const result = Contact.getDisplayName(contact)
+      expect(result).toEqual('Doran Martell')
+    })
+
+    it("should return the contact's primary email if no fullname", () => {
```
Comment (edas) [inline packages/cozy-doctypes/src/contacts/Contact.spec.js]: Are we sure of that? 

I would use fullname > givenName + familyName > email

Comment (Crash--) [reply packages/cozy-doctypes/src/contacts/Contact.spec.js]: This is the current specified behavior. We asked this morning to have a written specification about that. We'll update this function later if needed ;)

Comment (cedricmessiant) [reply packages/cozy-doctypes/src/contacts/Contact.spec.js]: I think we always have `fullname` if `givenName` and `familyName` are present but it costs almost nothing and it's obviously better. 
(We asked Thomas to clarify the order for the display name this morning)

**Comment by ptbrowne** [comment]
Great

**Pull request merged**